content/applications/marketing/email_marketing/lost_leads_email.rst

```diff
@@ -0,0 +1,337 @@
+:guilabel:`Lead/Opportunity` model from the drop-down menu.
+
+In the rules section, click the modify filter (:guilabel:`▶ (triangle pointing right)`) icon, below
+the :guilabel:`Recipients` field, to expand the filter rules. Leave the default
+:guilabel:`Blacklist` rule in place.
+
+Created on
+----------
+
+Begin by clicking and :guilabel:`New Rule` and select the :guilabel:`Created on` parameter in the
+new rule's first field, from the drop-down menu, to designate a specific time period during which
+the targeted leads were lost (i.e. 30 days prior, 90 days prior, previous year, etc.).
+
+Then, in the second field, select a date operator :guilabel:`<= (less than or equal to)`,
+:guilabel:`>= (greater than or equal to)`, or :guilabel:`is between` in order to frame the time
+selection chosen in the third field.
+
+In the third field, use the calendar popover window to select dates, and click :guilabel:`Apply` to
+lock in the time range.
+
+.. image:: lost_leads_email/created-on.png
+   :align: center
+   :alt: A custom filter rule setting the time period to be anything before today's date.
+
+.. important::
+   When there is more than one rule applied, make sure the statement at the top of the
+   :guilabel:`Recipients` filter list reads :guilabel:`Match all of the following rules`. If it does
+   not, click on the statement, and select :guilabel:`all` from the drop-down menu (as opposed to
+   :guilabel:`any`).
+
+   .. image:: lost_leads_email/match-all.png
+      :align: center
+      :alt: The statement at the top of the filters list, with the drop-down menu open.
+
+Stage
+-----
+
+Now add :guilabel:`Stage` filter to exclude leads in the *New*, *Qualified*, and *Won* stages of
+the sales pipeline.
+
+.. note::
+   This step assumes that the *New*, *Qualified*, and *Won* stages exist in the CRM pipeline,
+   however stage names may differ from business to business. Refer to the database's actual stage
+   names in the *CRM* app's pipeline to complete this step, accordingly.
+
+Begin again by clicking :guilabel:`New Rule` and selecting :guilabel:`Stage` from the first field's\
+drop-down menu. In the second field, select the :guilabel:`is not in` operator, and in the third
+field, select the :guilabel:`New`, :guilabel:`Qualified` and :guilabel:`Won` stages to define the
+rule's parameters.
+
+When the rule is added in this way, the logic in the third field renders as OR (`|`) statements.
+
+.. image::  lost_leads_email/stage-is-in.png
+   :align: center
+   :alt: Include multiple Stages in the filtering rule, using the "is in" operator
+
+.. tip::
+   Another way to add *Stage* rules, is to do so on a one-rule-per-row basis using the
+   :guilabel:`contains` or :guilabel:`does not contain` operators, and manually typing out the
+   defining characters in each stage name. This method, however, only allows for one selection at a
+   time, which can be useful for quickly turning on/off filters in the :guilabel:`Search...` bar.
+
```

Comment:
might be an extra line of space between the admonition and the image -- worth a quick double-check 👍

Reply:
Yup, removed it 👍